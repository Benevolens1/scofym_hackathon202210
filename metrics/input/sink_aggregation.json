{
	"1": {
		"type": "sink_aggregation",
		"arguments": {
            "header": {
                "identifier": "GTW02",
                "uploadTime" : 1664979668,
                "manufacturerName" : "Sagemcom"
            },
            "wifiData": [
                {
                    "eventTime" : 1664978268,
                    "deviceType": "PC",
                    "bytesSent": 72540,
                    "bytesReceived" : 150542,
                    "connection": "2_4GHZ",
                    "rssi": -40
                },
                {
                    "eventTime" : 1664978168,
                    "deviceType": "PC",
                    "bytesSent": 72540,
                    "bytesReceived" : 150542,
                    "connection": "2_4GHZ",
                    "rssi": -68
                },
                {
                    "eventTime" : 1664978268,
                    "deviceType": "PC",
                    "bytesSent": 72540,
                    "bytesReceived" : 150542,
                    "connection": "2_4GHZ",
                    "rssi": -76
                },
                {
                    "eventTime" : 1664978468,
                    "deviceType": "PC",
                    "bytesSent": 72540,
                    "bytesReceived" : 150542,
                    "connection": "5GHZ",
                    "rssi": -44
                },
                {
                    "eventTime" : 1664978568,
                    "deviceType": "PC",
                    "bytesSent": 72540,
                    "bytesReceived" : 150542,
                    "connection": "5GHZ",
                    "rssi": -48
                },
                {
                    "eventTime" : 1664978668,
                    "deviceType": "PC",
                    "bytesSent": 72540,
                    "bytesReceived" : 150542,
                    "connection": "5GHZ",
                    "rssi": -47
                }
                
            ]
        }
	},
	"2": {
		"type": "sink_aggregation",
		"arguments": {
            "header": {
                "identifier": "GTW01",
                "uploadTime" : 1664979989,
                "manufacturerName" : "Sagemcom"
            },
            "wifiData": [
                {
                    "eventTime" : 1664979456,
                    "deviceType": "IPhone",
                    "bytesSent": 87860,
                    "bytesReceived" : 896413,
                    "connection": "5GHZ",
                    "rssi": -84
                },
                {
                    "eventTime" : 1664979556,
                    "deviceType": "IPhone",
                    "bytesSent": 72540,
                    "bytesReceived" : 150542,
                    "connection": "5GHZ",
                    "rssi": -82
                },
                {
                    "eventTime" : 1664979656,
                    "deviceType": "IPhone",
                    "bytesSent": 72540,
                    "bytesReceived" : 150542,
                    "connection": "6GHZ",
                    "rssi": -78
                },
                {
                    "eventTime" : 1664979756,
                    "deviceType": "IPhone",
                    "bytesSent": 72540,
                    "bytesReceived" : 150542,
                    "connection": "2_4GHZ",
                    "rssi": -59
                },
                {
                    "eventTime" : 1664979856,
                    "deviceType": "IPhone",
                    "bytesSent": 72540,
                    "bytesReceived" : 150542,
                    "connection": "2_4GHZ",
                    "rssi": -62
                },
                {
                    "eventTime" : 1664979956,
                    "deviceType": "IPhone",
                    "bytesSent": 72540,
                    "bytesReceived" : 150542,
                    "connection": "2_4GHZ",
                    "rssi": -57
                }
                
            ]
        }
	}
}
