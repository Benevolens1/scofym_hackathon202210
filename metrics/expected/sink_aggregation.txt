1 {"identifier": "GTW02", "manufacturerName": "Sagemcom", "startTime": 1664978168, "endTime": 1664978668, "wifiAggregate": {"deviceType": "PC", "minRSSI": -76, "avgRSSI": -54, "maxRSSI": -40, "countBandChange": 1}, "anomalies_report": [{"eventTime": 1664978268, "deviceType": "PC", "rssi": -76, "connection": "2_4GHZ"}]}
2 {"identifier": "GTW01", "manufacturerName": "Sagemcom", "startTime": 1664979456, "endTime": 1664979956, "wifiAggregate": {"deviceType": "IPhone", "minRSSI": -84, "avgRSSI": -71, "maxRSSI": -57, "countBandChange": 2}, "anomalies_report": [{"eventTime": 1664979456, "deviceType": "IPhone", "rssi": -84, "connection": "5GHZ"}, {"eventTime": 1664979556, "deviceType": "IPhone", "rssi": -82, "connection": "5GHZ"}, {"eventTime": 1664979656, "deviceType": "IPhone", "rssi": -78, "connection": "6GHZ"}]}